coordinator/internal/logic/provertask/prover_task.go

```diff
@@ -91,3 +108,18 @@ func (b *BaseProverTask) checkParameter(ctx *gin.Context, getTaskParameter *coor
 	}
 	return &ptc, nil
 }
+
+func (b *BaseProverTask) getHardForkNumberByName(forkName string) (uint64, error) {
```

Comment:
For future reference, I believe we should make sure that multiple hard forks should not be scheduled on the same non-zero block height. Because there is no way we can figure out if a prover supports all of those hard forks or just some.

Reply:
If forks `A` and `B` happen on the same block `N`, I think the correct behaviour is this:
- If prover provides version `A`, then the block range will be empty (`height >= N && height < N`). This is basically a prover configuration error.
- If prover provides version `B`, then we can assign post-fork blocks to it (`height >= N`).

That said, I think we can keel only relevant forks in the monorepo fork config and remove all other forks (see my comment above).

Reply:
why A and B hard fork can happen at the block?

Reply:
An example is all those forks activated at `block: 0`.

But this probably won't happen for forks with height `> 0`.

Reply:
got it